libraries/SITL/examples/Webots_Python/controllers/ardupilot_vehicle_controller/webots_vehicle.py

```diff
@@ -155,14 +155,13 @@ def _handle_sitl(self, sitl_address: str = "127.0.0.1", port: int = 9002):
         Args:
             port (int, optional): Port to listen for SITL on. Defaults to 9002.
         """
-
         # create a local UDP socket server to listen for SITL
         s = socket.socket(socket.AF_INET, socket.SOCK_DGRAM) # SOCK_STREAM
         s.setsockopt(socket.SOL_SOCKET, socket.SO_REUSEADDR, 1)
         s.bind(('0.0.0.0', port))
 
         # wait for SITL to connect
-        print(f"Listening for ardupilot SITL (I{self._instance}) at 127.0.0.1:{port}")
+        print(f"Listening for ardupilot SITL (I{self._instance}) at {sitl_address}:{port}")
```

Comment:
Didn't we just bind 0.0.0.0?

I mean, the existing comment is wrong, but this one is also wrong.

I mean, *less* wrong :-)